src/main/java/pl/stormit/eduquiz/quizcreator/domain/question/Question.java

```diff
@@ -27,9 +30,12 @@ public class Question {
     @GeneratedValue
     private UUID id;
 
+    @NotBlank(message = "Content can't be empty")
+    @Size(min = 5, max = 25, message = "Content must be between 5 and 25 characters")
```

Comment:
I think, maximum value should be greater - e.g. we have quiz name "What was the name of the 1st capital of Poland?" in database.
Maybe we should use 255 characters?